### PR TITLE
Remove loading prior optimized parameters, update number nfev

### PR DESCRIPTION
**Pull request by jsnel**
Earlier optimization parameter files resulted in the wrong calibration of the maximum_number_function_evaluations

Follow up to PR #33  🧪 Fix number of 'maximum_number_function_evaluations'

### CODE REVIEW

File: pyglotaran_examples/ex_spectral_guidance/models/parameters_guidance.yml [hunk 1]
```diff
@@ -9,6 +9,7 @@ rates:
   - ["k4", 0.267]
   - ["k5", 0.0064]
   - ["k6", 1.3E-6]
+  - {"non-negative": True}
```
Comment (jsnel) [inline pyglotaran_examples/ex_spectral_guidance/models/parameters_guidance.yml:12]: This is needed else you get negative rates constants with enough iterations

File: pyglotaran_examples/study_fluorescence/target_analysis_script.py [hunk 2]
```diff
@@ -47,8 +47,10 @@
         model,
         parameter,
         {"dataset1": dataset},
-        maximum_number_function_evaluations=6,
+        maximum_number_function_evaluations=6,  # 6 for TRF, 46 for LM
+        # optimization_method="Levenberg-Marquardt", #lm needs nfev=46
         non_negative_least_squares=True,
+        # group=False
```
Comment (jsnel) [inline pyglotaran_examples/study_fluorescence/target_analysis_script.py:53]: Comment left as a reminder that at some point this was needed

File: pyglotaran_examples/test/simultaneous_analysis_3d_nodisp/sim_analysis_script_3d.py [hunk 3]
```diff
@@ -28,22 +28,15 @@
...
+parameters = load_parameters(parameters_path)
 print(model.validate(parameters=parameters))
 
 # define the analysis scheme to optimize
 scheme = Scheme(
     model,
     parameters,
     {"dataset1": dataset1, "dataset2": dataset2, "dataset3": dataset3},
-    maximum_number_function_evaluations=2,
+    maximum_number_function_evaluations=None,
```
Comment (jsnel) [inline pyglotaran_examples/test/simultaneous_analysis_3d_nodisp/sim_analysis_script_3d.py:39]: Takes 5 iterations for TRF

Comment (s-weigand) [reply pyglotaran_examples/test/simultaneous_analysis_3d_nodisp/sim_analysis_script_3d.py:39]: So why is it `None` and not 5?

**Review by s-weigand — APPROVED** [review]
Thanks for investigating, LGTM.

**Pull request merged**